lib/ansible/modules/network/frr/frr_facts.py

```diff
@@ -0,0 +1,470 @@
+#!/usr/bin/python
+#
```

Comment:
You can use the shorter version of license refer https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/network/junos/junos_command.py#L1